docs/publishing.md

```diff
@@ -10,23 +10,27 @@ sidebar_label: Publishing Packages
 
 1. Make changes and commit them to origin/branch
 2. Ensure your `git status` is porcelian
-3. On your local machine `npm run version`
+3. On your local machine
+   `npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease [--preid=<prerelease-id>] | from-git]`
```

Comment:
This isn't correct... why are you bypassing the lerna versioning script?

Reply:
Reverted